posts/renv-complications/index.qmd

```diff
@@ -0,0 +1,131 @@
+---
+title: "Things that can go wrong when using renv"
+author:
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-1476"
+date: "2023-09-26"
+categories: [R, reproducibility, renv]
+format:
+  html: 
+    toc: true
+---
+
+We use the [renv R package](https://rstudio.github.io/renv/) to ensure reproducibility of the training materials and the pipelines we are providing.
+But we sometimes get reports from users who struggle to rebuild the environment and run the code.
+
+In this post, we dissect the source of these issues, explain why in reality renv is not at fault, and how this is caused by the inherent complexity of reproducibility. renv documentation already includes [caveats](https://rstudio.github.io/renv/articles/renv.html#caveats) explaining why some situations are bound to require more complex tools. This blog post reiterates some of these caveats and illustrates them with concrete examples.
```

Comment:
```suggestion
In this post, I dissect the source of these issues, explain why in reality `renv` is not at fault, and how this is caused by the inherent complexity of reproducibility. The `renv` documentation already includes [caveats](https://rstudio.github.io/renv/articles/renv.html#caveats) explaining why some situations are bound to require more complex tools. This blog post reiterates some of these caveats and illustrates them with concrete examples.
```

Reply:
Also sticking to "we" even in the case of a single author post, to match what we've been doing previously.

```suggestion
In this post, we dissect the source of these issues, explain why in reality renv is not at fault, and how this is caused by the inherent complexity of reproducibility. The renv documentation already includes [caveats](https://rstudio.github.io/renv/articles/renv.html#caveats) explaining why some situations are bound to require more complex tools. This blog post reiterates some of these caveats and illustrates them with concrete examples.
```